apps/content/lib/paragraph/content_list.ex

```diff
@@ -202,6 +202,13 @@ defmodule Content.Paragraph.ContentList do
     |> combine()
   end
 
+  # If no types are specified, discard the :type key.
+  defp combine(%{type: [nil]} = ingredients) do
```

Comment:
are you sure you don't want to match `[]` (empty list) vs `[nil]` a list that contains a nil value?

Reply:
Thanks for bumping this. It was not a good idea to wrap the result of `content_type/1` in a list regardless of if it was nil or not, so I adjusted things to avoid the need for this extra function clause.